restli-server/src/test/pegasus/com/linkedin/restli/server/defaults/RecordB.pdl

```diff
@@ -0,0 +1,6 @@
+namespace com.linkedin.restli.server.defaults
+
+record RecordB {
+  field1: array[int] = [-1],
```

Comment:
Add usecase for array of records.

Reply:
Yeah, I think I responded this, usecase, in the newly added integration test.

Reply:
I think it is still good to add unit tests to cover the scenarios

Reply:
I have added the test for arrays in Bar/Foo related test cases, which is in the same test class but using the Bar.pdl and Foo.pdl